lib/speed.js

```diff
@@ -1,37 +1,129 @@
 'use strict';
 
-const rpad = (s, char, count) => (s + char.repeat(count - s.length));
-const lpad = (s, char, count) => (char.repeat(count - s.length) + s);
+const fs = require('fs');
+const { fork, execSync } = require('child_process');
+const makeResults = require('./speed/make-results.js');
+const PATH_TO_GET_TIME = __dirname + '/speed/get-time.js';
```

Comment:
You should use `path.join` function instead of string concatenation in such situations. This also applies to all the lines in this PR where concatenation is used when joining paths.